proto/campaign/query.proto

```diff
@@ -51,12 +52,22 @@ service Query {
     option (google.api.http).get = "/tendermint/spn/campaign/mainnet_vesting_account";
   }
 
+  // Queries a campaign summary
+  rpc CampaignSummary(QueryCampaignSummaryRequest) returns (QueryCampaignSummaryResponse) {
+    option (google.api.http).get = "/tendermint/spn/campaign/campaign_summary/{campaignID}";
+  }
+
+  // Queries a list of campaign summaries
+  rpc CampaignSummaries(QueryCampaignSummariesRequest) returns (QueryCampaignSummariesResponse) {
+    option (google.api.http).get = "/tendermint/spn/campaign/campaign_summary";
```

Comment:
might be misleading as an endpoint (due to the one above). Would change it to `/tendermint/spn/campaign/campaigns_summary`